addons/sale/models/sale_order_line.py

```diff
@@ -860,7 +860,7 @@ def _get_sale_order_line_multiline_description_variants(self):
         :return: the description related to special variant attributes/values
         :rtype: string
         """
-        if not self.product_custom_attribute_value_ids and not self.product_no_variant_attribute_value_ids:
+        if len(self.product_custom_attribute_value_ids) + len(self.product_no_variant_attribute_value_ids) <= 1:
             return ""
```

Comment:
checking that there is only one variant make sense to me, but if I set custom value "refursbished" with custom value manually set to "screen is still broken": I would expect to get this manually typed thing in the description I think.

So unless I'm missing something, I would expect to only ignore the case where there is a single non-variant custom value, eg:

```suggestion
        custom_ptavs = self.product_custom_attribute_value_ids.custom_product_template_attribute_value_id
        no_variant_ptavs = self.product_no_variant_attribute_value_ids._origin
        if not self.product_custom_attribute_value_ids and len(no_variant_ptavs - custom_ptavs) <= 1:
            return ""
        no_variant_ptavs = no_variant_ptavs[:0] if (no_variant_ptavs - custom_ptavs) <= 1 else no_variant_ptavs
```